tests/testthat/test-istr.R

```diff
@@ -327,16 +387,6 @@ test_that("a 0-row `scenarios` yields an error", {
   )
 })
 
-test_that("the thresholds are in the range 0 to 1", {
-  istr_arguments <- formals(istr_at_product_level)
-
-  low_threshold <- eval(istr_arguments$low_threshold)
-  high_threshold <- eval(istr_arguments$high_threshold)
-
-  expect_true(low_threshold >= 0 & low_threshold <= 1)
-  expect_true(high_threshold >= 0 & high_threshold <= 1)
-})
-
 test_that("NA in the reductions column yields `NA` in risk_category at product level", {
```

Comment:
@maurolepore I have excluded this test which ensures that the thresholds are in the range 0 and 1 because in this function:
``` r
threshold_categories <- function(data) {
  case_when(
    data == 2030 ~ list(low = 1/9, high = 2/9),
    .default = list(low = 1/3, high = 2/3)
  )
}
```
`.default` argument ensures that the thresholds will remain either `1/3` or `2/3`. Please feel free to ask me to  include this test again! Thanks!